### PR TITLE
Mark old storage classes as legacy

### PR DESCRIPTION
**Pull request by JesseLovelace**
Mark multi_regional and regional storage classes as legacy

### CODE REVIEW

**Comment by frankyn** [comment]
@quartzmo could you take a look at this? Thanks @JesseLovelace this LGTM.

**Comment by quartzmo** [comment]
Yes, will do.

File: google-cloud-storage/lib/google/cloud/storage/bucket.rb [hunk 1]
```diff
@@ -1137,7 +1138,8 @@ def file path, generation: nil, skip_lookup: nil, encryption_key: nil
         #   Determines how the file is stored and determines the SLA and the
         #   cost of storage. Accepted values include `:standard`, `:nearline`,
         #   and `:coldline`, as well as the equivalent strings returned by
-        #   {#storage_class}. For more information, see [Storage
+        #   {#storage_class}. `:multi_regional` and `:regional` are accepted
+        #   as legacy storage classes.For more information, see [Storage
         #   Classes](https://cloud.google.com/storage/docs/storage-classes)
```
Comment (quartzmo) [inline google-cloud-storage/lib/google/cloud/storage/bucket.rb]: Typo: .For

File: google-cloud-storage/lib/google/cloud/storage/bucket.rb [hunk 2]
```diff
@@ -357,8 +357,8 @@ def logging_prefix= logging_prefix
         ##
         # The bucket's storage class. This defines how objects in the bucket are
         # stored and determines the SLA and the cost of storage. Values include
-        # `MULTI_REGIONAL`, `REGIONAL`, `NEARLINE`, `COLDLINE`, and
-        # `DURABLE_REDUCED_AVAILABILITY`.
+        # `NEARLINE`, `COLDLINE`, and `DURABLE_REDUCED_AVAILABILITY`. `REGIONAL`
```
Comment (quartzmo) [inline google-cloud-storage/lib/google/cloud/storage/bucket.rb]: This appears to be inconsistent with the other lists below: 

"Values include `NEARLINE`, `COLDLINE`, and `DURABLE_REDUCED_AVAILABILITY`."

vs

 "Accepted values include `:standard`, `:nearline`, and `:coldline`."

**Comment by quartzmo** [comment]
Waiting to merge this PR before the next google-cloud-storage release.

**Comment by quartzmo** [comment]
@JesseLovelace Any chance of an update to this PR today? We are hoping to release google-cloud-storage soon.

**Pull request merged**